pyannote/audio/applications/change_detection.py

```diff
@@ -0,0 +1,411 @@
+
+    # -- SEQUENCE GENERATOR --
+    duration = config['sequences']['duration']
+    step = config['sequences']['step']
+
+    def saveSeg(filepath,filename,chn,segmentation):
+        f = open(filepath,'w')
+        for idx, val in enumerate(segmentation):
+                line = filename+' '+str(idx)+' '+str(chn)+' '+str(int(val[0]*100))+' '+str(int(val[1]*100-val[0]*100))+'\n'
+                f.write(line)
+        f.close()
+
+    def get_aggregation(epoch):
+        weights_h5 = WEIGHTS_H5.format(epoch=epoch)
+        sequence_labeling = SequenceLabeling.from_disk(
+                architecture_yml, weights_h5)
+
+
+        aggregation = SequenceLabelingAggregation(
+                sequence_labeling, feature_extraction,
+                duration=duration, step=step)
+
+        return aggregation
+
+
+    filepath = store_dir+'/'+str(threshold) +'/'
+    os.makedirs(filepath)
+
+    aggregation = get_aggregation(nb_epoch-1)
+
+    predictions = {}
+    for dev_file in getattr(protocol, subset)():
+        uri = dev_file['uri']
+        predictions[uri] = aggregation.apply(dev_file)  
+
+    # initialize peak detection algorithm
+    peak = Peak(alpha=threshold, min_duration=2.5)
+
+    for dev_file in getattr(protocol, subset)():
+        uri = dev_file['uri']
+        prediction = predictions[uri]
+        hypothesis = peak.apply(prediction)
+        filepath = store_dir+'/'+str(threshold) +'/'+uri+'.0.seg'
+        chn = 1
+        saveSeg(filepath,uri,chn,hypothesis)
+
+
+if __name__ == '__main__':
+
+    arguments = docopt(__doc__, version='Speaker change detection')
+    db_yml = os.path.expanduser(arguments['--database'])
+    preprocessors = {'wav': FileFinder(db_yml)}
+
+    if '<database.task.protocol>' in arguments:
+        protocol = arguments['<database.task.protocol>']
+        database_name, task_name, protocol_name = protocol.split('.')
+        database = get_database(database_name, preprocessors=preprocessors)
+        protocol = database.get_protocol(task_name, protocol_name, progress=True)
+
+    subset = arguments['--subset']
+
+    if arguments['train']:
+        experiment_dir = arguments['<experiment_dir>']
+        if subset is None:
+            subset = 'train'
+        train_dir = experiment_dir + '/train/' + arguments['<database.task.protocol>'] + '.' + subset
+        train(protocol, experiment_dir, train_dir, subset=subset)
+
+
+    if arguments['evaluate']:
+        train_dir = arguments['<train_dir>']
+        if subset is None:
+                subset = 'development'
+        store_dir = train_dir + '/evaluate/' + arguments['<database.task.protocol>'] + '.' + subset
+        res = evaluate(protocol, train_dir, store_dir, subset=subset)
+
+
+    if arguments['apply']:
+        train_dir = arguments['<train_dir>']
+        if subset is None:
+            subset = 'development'
+        threshold = arguments['--threshold']
+        if threshold is None:
+            threshold = 0.7
```

Comment:
You can set the default value directly in the docstring.
See, for example, https://github.com/pyannote/pyannote-video/blob/master/scripts/pyannote-face.py#L57

This also has the advantage of telling the user what the default value is :)